src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Don't reset margins and paddings using (*) selector. It applies to all elements, which is not good for perfomance.
Better specify the exact elements